rbac/api/cross_access/view.py

```diff
@@ -47,6 +53,11 @@
 class CrossAccountRequestFilter(filters.FilterSet):
     """Filter for cross account request."""
 
+    def org_id_filter(self, queryset, field, values):
```

Comment:
I know we're not using it on the current `account_filter` method, but I think we could make use of this shared filter here: https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/management/filters.py#L38

Reply:
Do you mean how the group objects UUID filter is setup? https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/management/group/view.py#L72

Reply:
Yeah, similar to that.